### PR TITLE
Remove GHT support

### PR DESCRIPTION
**Pull request by pblottiere**
Hi everyone,

This PR follows this [thread](https://lists.osgeo.org/pipermail/pgpointcloud/2018-June/000289.html) saying that `GHT is dead. It's a bad fit for the db environment.`.

So I propose to remove the GHT support from pointcloud. Do you agree?

If this PR is merged, I'll take care about updating the [PDAL plugin](https://github.com/PDAL/PDAL/tree/master/plugins/pgpointcloud/io) accordingly.

Moreover, the next issues should be closed : 
- https://github.com/pgpointcloud/pointcloud/issues/35
- https://github.com/pgpointcloud/pointcloud/issues/196

### CODE REVIEW

**Comment by elemoine** [comment]
If we remove GHT support we'll at least need to do the same in PDAL.

**Comment by pblottiere** [comment]
> If we remove GHT support we'll at least need to change PDAL's pgpointcloud writer to reflect that change.

Indeed! That's what I meant by saying : 

> If this PR is merged, I'll take care about updating the PDAL plugin accordingly.

**Comment by elemoine** [comment]
Sorry I hadn't noticed your comment. We're inline.

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +25,7 @@ addons:
 
 script:
   - eclint check * */* */cunit/*
-  - ./tools/build-install.sh --with-lazperf=/usr/local --with-libght=/usr/local && make check
-  - ./tools/build-install.sh --with-lazperf=/usr/local --without-libght && ./tools/valgrind.sh
+  - ./tools/build-install.sh --with-lazperf=/usr/local && make check
+  - ./tools/build-install.sh --with-lazperf=/usr/local && ./tools/valgrind.sh
```
Comment (elemoine) [inline .travis.yml]: Now that we don't build with libght we no longer need to run build-install.sh twice.

Comment (pblottiere) [reply .travis.yml]: Fixed

**Comment by pblottiere** [comment]
Thanks for your review @elemoine 

> I'd like that we tag v1.1.1 before merging this. And we could create v1.2.0 when the GHT support removal is merged.

I fully agree :+1:!

**Comment by elemoine** [comment]
Let's merge this.

**Pull request merged**